sixtyfps_runtime/corelib/model.rs

```diff
@@ -462,56 +462,50 @@ impl Model for bool {
     }
 }
 
-/// Properties of type array in the .60 language are represented as
-/// an [`Option`] of an [`Rc`] of something implemented the [`Model`] trait
+/// Properties of type Array in the .60 language are represented as
+/// a `SharedModel` that implements the `Model` trait.
+#[repr(transparent)]
 #[derive(derive_more::Deref, derive_more::DerefMut, derive_more::From, derive_more::Into)]
-pub struct ModelHandle<T>(pub Option<Rc<dyn Model<Data = T>>>);
+pub struct SharedModel<T>(Option<Rc<dyn Model<Data = T>>>);
 
-impl<T> core::fmt::Debug for ModelHandle<T> {
-    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
-        write!(f, "ModelHandle({:?})", self.0.as_ref().map(|_| "dyn Model"))
+impl<T> std::fmt::Debug for SharedModel<T> {
```

Comment:
```suggestion
impl<T> core::fmt::Debug for SharedModel<T> {
```

We want to compile without std on MCU, remember